common/services/github.js

```diff
@@ -50,7 +50,7 @@ async function _getReviewsFromGithub(pull_number){
     owner,
     repo,
     pull_number
-  });
+  }); 
```

Comment:
```suggestion
  });
```
un espace en trop s'est glissé